future-apps/actor/test/vault.js

```diff
@@ -0,0 +1,238 @@
+// Test that Actor is a fully functioning Vault by running the same tests against the Actor app
```

Comment:
Copy pasting this code from the Vault tests feels dirty. @sohkai suggestions on how to do this in a clean way?